src/OI.h

```diff
@@ -45,16 +34,35 @@ class OI {
 	std::shared_ptr<JoystickButton> dButtonB;
 	std::shared_ptr<JoystickButton> dButtonA;
 
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
 public:
 	OI();
-
-	// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=PROTOTYPES
-
 	std::shared_ptr<Joystick> getdriver();
 	std::shared_ptr<Joystick> getcoDriver();
 
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=PROTOTYPES
+	static const int kButtonA_ID = 1;
+	static const int kButtonB_ID = 2;
+	static const int kButtonX_ID = 3;
+	static const int kButtonY_ID = 4;
+	static const int kButtonLB_ID = 5;
+	static const int kButtonRB_ID = 6;
+	static const int kButtonBack_ID = 7;
+	static const int kButtonStart_ID = 8;
+	static const int kButtonL3_ID = 9;
+	static const int kButtonR3_ID = 10;
+
+	static const int kLGreen_ID = 15;
+	static const int kLRed_ID = 12;
+	static const int kLBlack_ID = 8;
+	static const int kLWhite_ID = 13;
+	static const int kLBlue_ID = 9;
+	static const int kRGreen_ID = 4;
+	static const int kRBlack_ID = 3;
+	static const int kRRed_ID = 1;
+	static const int kRYellow_ID = 2;
```

Comment:
I assume this is not random chaos? Just thinking of ways to organize them to make it easier to find